nucliadb/nucliadb/ingest/tests/servicer/test_knowledgebox_servicer.py

```diff
@@ -89,6 +89,31 @@ async def test_create_knowledgebox(
     assert len(resp.json()["data"]) == expected_spans
 
 
+@pytest.mark.asyncio
+async def test_create_knowledgebox_with_similarity(grpc_servicer: IngestFixture):
+    stub = writer_pb2_grpc.WriterStub(grpc_servicer.channel)
+
+    pb = knowledgebox_pb2.KnowledgeBoxNew(slug="test-dot")
+    pb.config.title = "My Title"
+    pb.config.similarity = utils_pb2.VectorSimilarity.Dot
+    result = await stub.NewKnowledgeBox(pb)  # type: ignore
+    assert result.status == knowledgebox_pb2.KnowledgeBoxResponseStatus.OK
+
+    kbid = knowledgebox_pb2.KnowledgeBoxID(slug="test-dot")
+    resp = await stub.GetKnowledgeBox(kbid)  # type: ignore
+    assert resp.config.similarity == utils_pb2.VectorSimilarity.Dot
+
+    # Check that by default we get cosine similarity
```

Comment:
Split in two tests, as there are two things under tests (set specific similarity and show the default)